recipes/tuiview/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "1.1.8" %}
+
+package:
+    name: tuiview
+    version: {{ version }}
+
+source:
+    fn: TuiView-{{ version }}.tar.gz  # [unix]
+    fn: TuiView-{{ version }}.zip  # [win]
+    url: https://bitbucket.org/chchrsc/tuiview/downloads/TuiView-1.1.8.tar.gz  # [unix]
+    url: https://bitbucket.org/chchrsc/tuiview/downloads/TuiView-1.1.8.zip  # [win]
```

Comment:
Unless they include different files, line endings don't matter.  You can keep just the tarball and things should work